### PR TITLE
[Form] Bootstrap 5 documentation

### PR DESCRIPTION
**Pull request by ker0x**
This PR introduce documentation for [Bootstrap 5 form template](https://github.com/symfony/symfony/pull/39157)

### CODE REVIEW

File: form/bootstrap5.rst
Comment (t-richard) [inline form/bootstrap5.rst]:
```suggestion
Bootstrap 5 allows to render checkboxes as "`switches`_". You can enable this feature
```

File: form/bootstrap5.rst
Comment (t-richard) [inline form/bootstrap5.rst]:
```suggestion
    Switches only work with **checkbox**.
```

File: form/bootstrap5.rst
Comment (t-richard) [inline form/bootstrap5.rst]: Passive voice is harder to read, especially for non-native speakers, so I would write
```suggestion
    If you fill the `help` option of your form, it will also be rendered
```

File: form/bootstrap5.rst
Comment (t-richard) [inline form/bootstrap5.rst]:
```suggestion
    work properly.
```

**Review by t-richard — CHANGES_REQUESTED** [review]
Thank you for this PR and the associated feature PR :pray: 

Just some minor English errors I spotted.

**Comment by ker0x** [comment]
Thank you @t-richard for your review !

File: form/bootstrap5.rst
Comment (OskarStark) [inline form/bootstrap5.rst]:
```suggestion
    on the ``label_attr`` option::
```

Comment (ker0x) [reply form/bootstrap5.rst]: @OskarStark Thanks for the solution, I had trouble understanding the error

**Comment by javiereguiluz** [comment]
Romain, thanks for this contribution!

We'll need to wait until the code PR is merged ... that will need to wait until Bootstrap 5 is released as stable.

When this happens, we'll need to reevaluate if we add this new doc article or transform the existing "bootstrap4" doc article into a generic article about all Bootstrap versions. We'll probably do that if differences between versions are not many, because that makes things easier for readers. Thanks.

**Comment by ker0x** [comment]
Hello @javiereguiluz 

Now that the code PR has been merged and the release of 5.3 is coming, have you any feed back regarding this PR ?

**Comment by javiereguiluz** [comment]
Romain, thanks for the ping! After thinking about this, I've merged your PR "as is". There are some differences between Bootstrap 4 and 5, so it might be more clear for the reader to separate them. Thanks!

**Pull request merged**
